nomad-cluster-security.md

```diff
@@ -0,0 +1,14 @@
+# Nomad Cluster Security
+
+1. What is the noamd cluster size?
+1. Is Nomad deployed single region or multi-region? What is the deployment strategy? How is the high availability and failover designed? How is leader and follower selected?
+1. Is Nomad integrated with consul? How the DNS resolutions works to connect to the consul cluster?
+1. How is Nomad to Consul connectivity? Is it over HTTP secured with TLS? And is there any consul token to encrypt the traffic?
+1. Is the replication happens between the Nomad cluster in different regions?
+1. Does Nomad cluster uses the ACL policies, Tokens and sentinal policies?
+1. Do we have bootstrap Nomad ACL systems? it will make sure we have tokens , policies - rules and capabilities
+1. Are you using vault for secret management? Is your vault has the version support?
```

Comment:
Please fix grammar.

Reply:
Fixed the grammar error :- 

Is the replication between the Nomad cluster in different regions?
Does Nomad cluster use the ACL policies, tokens and sentinel policies?
Do we have bootstrap Nomad ACL systems? It will make sure we have tokens , policies - rules and capabilities.
Are you using a vault for secret management? Does your vault have the version support?